hermes/probe/read/read_test.go

```diff
@@ -32,15 +17,16 @@ import (
 )
 
 const (
-	fileSizeBytes = 100
+	fileSizeBytes     = 100
+	readTestProbeName = "read_test_probe"
 )
 
 func TestReadFile(t *testing.T) {
 	target := &probe.Target{
 		&probepb.Target{
 			Name:                   "hermes",
 			TargetSystem:           probepb.Target_GOOGLE_CLOUD_STORAGE,
-			TotalSpaceAllocatedMib: int64(1000),
+			TotalSpaceAllocatedMib: int64(1),
```

Comment:
Is the `int()` required? Wouldn't just `1` work?

Reply:
it works!! so changed